net-analyzer/netdata/netdata-1.6.0.ebuild

```diff
@@ -0,0 +1,104 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python{2_7,3_4,3_5} )
```

Comment:
If you merge https://github.com/gentoo/gentoo/pull/4284 then I can update this PR